src/jit/compiler.h

```diff
@@ -7770,7 +7770,13 @@ XXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX
 
     bool isSIMDClass(CORINFO_CLASS_HANDLE clsHnd)
     {
-        return info.compCompHnd->isInSIMDModule(clsHnd);
+        if (isIntrinsicType(clsHnd))
```

Comment:
It is unfortunate that `isIntrinsicType` calls `getClassAttribs`. `getClassAttribs` does a lot more. Would it make sense to rename `isInSIMDModule` on JIT/EE interface to `isIntrinsicType` ?

Reply:
I'd be fine with doing so, I'll update after I get back home tonight :)

Reply:
Done. I think this makes the overall logic quite a bit simpler. We were already caching `isIntrinsicType` in the VM, so this reduces the cost of checking elsewhere.